Classes/UT3DmgType_HellbenderRearGunBeam.uc

```diff
@@ -0,0 +1,11 @@
+class UT3DmgType_HellbenderRearGunBeam extends VehicleDamageType
+	abstract;
+
+defaultproperties
+{
+     VehicleClass=Class'UT3HellbenderRearGunPawn'
+     DeathString="%o was disintegrated by %k's Hellbender turret beam."
+     FemaleSuicide="%o somehow managed to disintegrate herself with a Hellbender turret."
+     MaleSuicide="%o somehow managed to disintegrate himself with a Hellbender turret."

```

Comment:
I'd rather these were stock UT3 messages. Sure, they may not be quite as precise, but I'd rather stick with showing off how they actually look in UT3.

Reply:
Alright, what about the more lame suicide though, UT2004 is more interesting on that most of the time

Reply:
I mean really how do you shoot yourself with a turret?  For that matter a death message in UT2004 I really love but have never seen in-game is the warp space-time and sniped them self now that's funny and thoughtful

Reply:
Keep to the UT3 ones. Indeed it's pretty much impossible to suicide like that, hence keep the original wording.